src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions/FuncMemoizeExtensions.cs

```diff
@@ -191,6 +191,35 @@ public static Func<T> AsMemoized<T>(this Func<T> func)
 			};
 		}
 
+		/// <summary>
+		/// Memoizer with six parameters, used to perform a lazy-cached evaluation. (see http://en.wikipedia.org/wiki/Memoization)
+		/// </summary>
+		/// <typeparam name="TParam1">The first parameter type to memoize</typeparam>
+		/// <typeparam name="TParam2">The second parameter type to memoize</typeparam>
+		/// <typeparam name="TParam3">The third parameter type to memoize</typeparam>
+		/// <typeparam name="TParam4">The fourth parameter type to memoize</typeparam>
+		/// <typeparam name="TParam5">The fifth parameter type to memoize</typeparam>
+		/// <typeparam name="TParam6">The sixth parameter type to memoize</typeparam>
+		/// <param name="func">the function to evaluate</param>
+		/// <returns>A memoized value</returns>
+		public static Func<TParam1, TParam2, TParam3, TParam4, TParam5, TParam6, TResult> AsMemoized<TParam1, TParam2, TParam3, TParam4, TParam5, TParam6, TResult>(this Func<TParam1, TParam2, TParam3, TParam4, TParam5, TParam6, TResult> func)
```

Comment:
Why do you need this in this PR ?

Reply:
@jeromelaban In UIFontHelper.iOS.cs, `_tryGetFont` is now 6 parameters, and we call `AsMemoized` on it.

Reply:
Ah! Missed it. Thanks. Maybe we should be using a record as the parameter then?

Reply:
We could. We could also use a tuple like in:

https://github.com/unoplatform/uno/blob/d83450c5a1559f2532b7b2c00f77edfd9b2b0b92/src/Uno.UI/Controls/UIStringAttributesHelper.iOS.cs#L14-L27

Let me know your preference.

Reply:
A tuple can work as well, though having a record makes for less duplication (you'll have to specify the Tuple at least twice).